### PR TITLE
Issue 34 - Licensing Documentation

### PR DESCRIPTION
**Pull request by cgbosse**
docs: Licensing documentation

Updated the license file with the code license and included the licensing folder with the individual licenses md files, and the contributor agreement from the Chinese Ideographs repository.

What will need to be done is to update the Readme file to reflect the licensing complexity as recommended by [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository).

As of now I have included the code license reference in the license file for the repository, the AGPL-3.0.

### CODE REVIEW

**Comment by cgbosse** [comment]
I fixed one link problem and I reviewed the megalinter result:
The other mistakes are no mistakes.
They are a list of unknown words or abbreviations in several documents.

imagen

**Comment by josecelano** [comment]
hi @cgbosse at the end of the mega-liter error log file you can find the solution for cspell:

```
Results of cspell linter (version 5.13.2)
See documentation on https://megalinter.github.io/descriptors/spell_cspell/
-----------------------------------------------

❌ [ERROR] for workspace /github/workspace
Linter raw log:
 1/73 ./.bandit.yml 736.59ms
...
40/73 ./licensing/cc-by-sa.md 91.32ms X
/github/workspace/licensing/cc-by-sa.md:23:60 - Unknown word (creativecommons)
/github/workspace/licensing/cc-by-sa.md:23:80 - Unknown word (compatiblelicenses)
/github/workspace/licensing/cc-by-sa.md:49:129 - Unknown word (sublicensable)
/github/workspace/licensing/cc-by-sa.md:77:204 - Unknown word (waivable)
/github/workspace/licensing/cc-by-sa.md:169:590 - Unknown word (creativecommons)
/github/workspace/licensing/cc-by-sa.md:171:40 - Unknown word (creativecommons)
41/73 ./licensing/contributor_agreement_v01.md 66.04ms X
/github/workspace/licensing/contributor_agreement_v01.md:7:120 - Unknown word (NCCA)
/github/workspace/licensing/contributor_agreement_v01.md:11:5 - Unknown word (NCCA)
/github/workspace/licensing/contributor_agreement_v01.md:13:5 - Unknown word (NCCA)
/github/workspace/licensing/contributor_agreement_v01.md:13:130 - Unknown word (subpoints)
/github/workspace/licensing/contributor_agreement_v01.md:13:157 - Unknown word (NCCA)
/github/workspace/licensing/contributor_agreement_v01.md:21:97 - Unknown word (sublist)
/github/workspace/licensing/contributor_agreement_v01.md:69:268 - Unknown word (sublist)
/github/workspace/licensing/contributor_agreement_v01.md:90:5 - Unknown word (Oberhachingerstr)
/github/workspace/licensing/contributor_agreement_v01.md:91:10 - Unknown word (Grünwald)
...
CSpell: Files checked: 73, Issues found: 16 in 3 files


You can skip this misspellings by defining the following .cspell.json file at the root of your repository
Of course, please correct real typos before :)

{
    "version": "0.1",
    "language": "en",
    "ignorePaths": [
        "**/node_modules/**",
        "**/vscode-extension/**",
        "**/.git/**",
        ".vscode",
        "package-lock.json",
        "report"
    ],
    "words": [
        "Gr\u00fcnwald",
        "NCCA",
        "Oberhachingerstr",
        "compatiblelicenses",
        "creativecommons",
        "sublicensable",
        "sublist",
        "subpoints",
        "waivable"
    ]
}


You can also copy-paste report/.cspell.json at the root of your repository
```

I've just [fixed it](https://github.com/Nautilus-Cyberneering/nautilus-librarian/pull/37/commits/75093f6fc1402c76b9d6c21122fac11d7491c8c7). 

The project dictionary is this [here](https://github.com/Nautilus-Cyberneering/nautilus-librarian/blob/main/.cspell.json#L13).

Regarding the other failing workflows (test*integration*) the reason is the same as in this [PR](https://github.com/Nautilus-Cyberneering/nautilus-librarian/pull/36). I'm going to apply the same patch in that a new issue and you can rebase this branch when it's fixed.

File: licensing/file_header_agplv3.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+    Chinese Ideographs
```
Comment (da2ce7) [inline licensing/file_header_agplv3.txt:1]: Update to Nautilus Librarian

File: licensing/file_header_agplv3.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    
+    Project owner: Nautilus Cyberneering GmbH.
+    Github repository: https://github.com/Nautilus-Cyberneering/chinese-ideographs
```
Comment (da2ce7) [inline licensing/file_header_agplv3.txt:4]: Same Here.

File: licensing/file_header_agplv3.txt (hunk 1 above)
Comment (cgbosse) [reply licensing/file_header_agplv3.txt:1]: Corrected.

**Review by cgbosse — COMMENTED** [review]
Corrected link to the librarian repository.

File: licensing/file_header_agplv3.txt (hunk 2 above)
Comment (cgbosse) [reply licensing/file_header_agplv3.txt:4]: Corrected.

**Comment by cgbosse** [comment]
The error is nothing but a link to a new folder that will be created by this PR.
imagen

**Pull request merged**